src/account/PluginManager2.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.25;
+
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import {IPlugin} from "../interfaces/IPlugin.sol";
+import {FunctionReference} from "../interfaces/IPluginManager.sol";
+import {FunctionReferenceLib} from "../helpers/FunctionReferenceLib.sol";
+import {AccountStorage, getAccountStorage, toSetValue} from "./AccountStorage.sol";
+
+abstract contract PluginManager2 {
```

Comment:
Perhaps consider giving it a more specific name :)

Reply:
We can, but these are temporary install paths - see the comment here: https://github.com/erc6900/reference-implementation/blob/adam/default-validation/src/interfaces/IPluginManager.sol#L25